users/writer_profiles/writersProfileModel.js

```diff
@@ -0,0 +1,57 @@
+const db = require("../../knex/knex");
+
+module.exports = {
+    add,
+    findWritersProfile,
+    findWriterProfilBy,
+    findWriterProfileById,
+    findByUserType,
+    findByEmail,
+    getItemsByUserId,
+    updateWriterProfile,
+    deleteWriteProfile,
+};
+
+function getItemsByUserId(id) {
+    return db("items").where({ "items.user_id": id });
+}
+
+//returns all writers user type profiles
+function findWritersProfile() {
+    const query = db("writer_profiles").select(
+        "id",
+        "first_name",
+        "last_name",
+        "email",
+        "user_type"
+    );
+    return query;
+}
+
+//returns specific writer  profile or null
+function findWriterProfileById(id) {
+    return db("writer_profiles").where({ id }).first();
+}
+//returns writer  profile by filter
+function findWriterProfilBy(filter) {
+    return db("writer_profiles").where(filter).first();
+}
+
+async function add(user) {
```

Comment:
From this line down, should we separate these functions into a separate userModel.js file so that both user types can use the same functions for adding and logging in? That way we don't have to repeat the code anywhere?